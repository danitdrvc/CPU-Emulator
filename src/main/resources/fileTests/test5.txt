NOT QWORD R0
XOR QWORD R1, R0
CMP QWORD R2, R0
HALT