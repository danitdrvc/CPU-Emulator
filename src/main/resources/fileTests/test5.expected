[-11, -16, 0, 1]